lib/node_modules/@stdlib/math/base/special/expm1rel/docs/repl.txt

```diff
@@ -21,9 +21,15 @@
 
     Examples
     --------
-    > var TODO = {{alias}}()
-    TODO: abbrev output
-
+    > var y = {{alias}}( 0.0 )
+    ~1.0
+	> y = {{alias}}( 1.0 )
```

Comment:
The indentation is off here. Each line should have 4-space indentation. @Planeshifter has written a linter to catch such mistakes, but it is still a WIP and has not been enabled yet.

Reply:
I'm not seeing where the indentation is off.

Reply:
Ah...there are tabs. In REPL.txt, we use space indentation. If you look closely here on the PR, you will see tabs used starting at L26.

Reply:
Ok, you mean non-tab.  Sorry.

Reply:
Correct. My apologies for not being more clear.